R-package/R/lgb.Booster.R

```diff
@@ -469,8 +480,19 @@ Booster <- R6::R6Class(
                        predcontrib = FALSE,
                        header = FALSE,
                        reshape = FALSE,
+                       params = list(),
                        ...) {
 
+      additional_params <- list(...)
+      if (length(additional_params) > 0L) {
+        warning(paste0(
+          "Booster$predict(): Found the following passed through '...': "
+          , paste(names(additional_params), collapse = ", ")
+          , ". These will be used, but in future releases of lightgbm, this warning will become an error. "
+          , "Add these to 'params' instead. See ?predict.lgb.Booster for documentation on how to call this function."
```

Comment:
> See ?predict.lgb.Booster for ...

Sorry, I'm not familiar with this, but it looks like a copy-paste typo from `predict.lgb.Booster`.

Reply:
ah, good question! Nope, this one was not a mistake.

Since the `R6` class `Booster` is not currently exported https://github.com/microsoft/LightGBM/blob/11d7608f2d755899a88bfa1c0fbc779b9e9b3505/R-package/NAMESPACE, users of the R package cannot view any documentation for it with something like `?Booster`.

So I think that in the current state of the package, `?predict.lgb.Booster` is the best place to direct them.

Reply:
Ah, OK, thanks a lot for the explnation!